haas/ext/switches/migrations/mock/b5b31d19257d_rename_mockswitch_table_for_flask_.py

```diff
@@ -7,17 +7,16 @@
 Create Date: 2016-03-22 05:11:19.585905
 
 """
+from alembic import op
```

Comment:
I am not very positive but I assume it is fine - moving them to the top of the file. 

//cc @henn